gamemodes/terrortown/gamemode/server/sv_main.lua

```diff
@@ -842,7 +843,7 @@ local function NameChangeKick()
         ---
         -- @realm server
         -- stylua: ignore
-        if not ply.has_spawned or ply.spawn_nick == ply:Nick() or hook.Run("TTTNameChangeKick", ply) then continue end
+        if ply:IsBot() or not ply.has_spawned or ply.spawn_nick == ply:Nick() or hook.Run("TTTNameChangeKick", ply) then continue end
```

Comment:
you should mention this change, I guess it helps with bot addons?

Reply:
is this also an upstream change?

Reply:
No idea, if it is then could you please link the commits / pullrequests so it is easier to get what is done here? thanks!

Reply:
The reason this is changed is because player.Humans does not use the iterator cache so by introducing it to this condition it is effectually the same as it was before